src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -494,22 +460,7 @@ ChannelsStatusResponse getChannelsStatus(
       request.setChannels(requestDTOs);
       request.setRole(this.role);
 
```

Comment:
pl remove objectMapper from this class completely, nobody else is using it after your change!